numpy/f2py/backends/src/meson.build.src

```diff
@@ -0,0 +1,22 @@
+project('${modulename}', 'c',
+  version : '0.1',
```

Comment:
I think you want to set the minimum `meson` version here, and it should be very recent; lots of Python bugs were fixed recently. I recommend the most recent release, `0.63.2`, and bumping it up to `0.63.3`/`0.64.0` when those become available.

Reply:
Corrected it.